Remotion/Core/Extensions.UnitTests/Logging/Log4NetTests/Log4NetLoggerProviderTest.cs

```diff
@@ -31,7 +30,6 @@ public void CreateLogger_WithValidCategoryName_ReturnsLoggerInstance ()
 
     var logger = loggerProvider.CreateLogger(categoryName);
 
-    Assert.That(logger, Is.Not.Null);
     Assert.That(logger, Is.InstanceOf<Log4NetLogger>());
   }
```

Comment:
assertion for the log4net logger's name matching categoryName

Reply:
It's okay to export the log4net Logger on the public API.

Naming: Logger.

Checkout Log4NetLogManagerTest.GetLogger_WithNameAsString_ReturnsSameLoggerTwice on how we solved this in the original code base